### DISCUSSION TITLE
Better rendering of rows and columns in markdown

### DISCUSSION THREAD

**Discussion by nonetrix** [fmaclen/hollama#207] [Ideas]
Maybe mildly opinionated, so I have put this in discussions as a idea instead. But I am not a fan of how rows and columns render in this UI currently. They are visually separated enough in my personal opinion, here is what they look like:
![image](https://github.com/user-attachments/assets/ca6828dd-ac44-4a42-be49-f1e10279c910)
And here is ChatGPT
![image](https://github.com/user-attachments/assets/97f2a620-723b-4e0a-9da8-375416818cf2)

**Comment by fmaclen** [comment]
Oh, yeah hehe... you actually found a bug 😅 
That's not an intentional design choice, those tables were never styled.

I just added a new issue #210 so we can get it fixed.

**Reply by fmaclen** [reply]
It's fixed as of [`0.17.4`](https://github.com/fmaclen/hollama/releases/tag/0.17.4)
